components/behaviors/time-series.schema.json

```diff
@@ -44,6 +44,8 @@
             "advertising.thirdQuartiles": "Advertising Third Quartiles",
             "web.webpagedetails.pageViews": "Web Webpagedetails Page Views",
             "web.webinteraction.linkClicks": "Web Webinteraction Link Clicks",
+            "web.fillOutForm": "Web Fill Out Form",
+            "web.webInteraction.clickLink": "Web Webinteraction Click Link",
```

Comment:
1. Should we reuse "web.webinteraction.linkClicks" instead?
2. The path name was in all lower cases from the existing ones. The new ones seemed to use camel case. Is there a consistent issue? On the published wiki, we seemed to use lowercase!